src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -319,7 +319,7 @@ export default class AircraftController {
      */
     aircraft_remove_all() {
         for (let i = 0; i < this.aircraft.list.length; i++) {
-            this.removeStripView(this.aircraft.list[i]);
+            this.aircraft_remove(this.aircraft.list[i]);
         }
 
         this.aircraft.list = [];
```

Comment:
This should also be removed from remove_all(), because remove() will take the aircraft out of the aircraft list (via `.removeAircraftModelFromList()`)

Reply:
done